frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -13,6 +13,7 @@ local T = require("ffi/util").template
 local ReaderDictionary = InputContainer:new{
     data_dir = nil,
     dict_window_list = {},
+    lookup_msg = _("Dictionary lookup of :\n“%1” ...")
```

Comment:
It's grammatical, but you're probably looking for something like dictionary result or definition instead? NB I haven't really checked when this would show up or anything.